data/src/main/java/org/apache/iceberg/data/parquet/GenericParquetReaders.java

```diff
@@ -380,21 +383,21 @@ private FixedReader(ColumnDescriptor desc) {
   }
 
   static class RecordReader extends StructReader<Record, Record> {
-    private final StructType struct;
+    private final StructType record;
```

Comment:
@rdsr, same comment about naming here. Sorry I didn't point it out the first time.

Reply:
np. Will update

Reply:
updated. Thks!